emukit/quadrature/acquisitions/mutual_information.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+from typing import Tuple
+
+from emukit.core.acquisition import Acquisition
```

Comment:
I think we should use relative imports for imports from emukit, but maybe @apaleyes can clarify?

Reply:
I think you are right. I will change that.